crates/uv-interpreter/python/get_interpreter_info.py

```diff
@@ -520,6 +520,7 @@ def main() -> None:
         "prefix": sys.prefix,
         "base_executable": getattr(sys, "_base_executable", None),
         "sys_executable": sys.executable,
+        "sys_path": sys.path,
```

Comment:
is it desirable for cwd to be in here? (if not, it should be removed, accounting for sys.flags.safe_path)

Reply:
probably will leave this value as is so that it still equals to `sys.path` (it may be confusing if it's called `sys_path` but some filtering was applied beforehand) - but added filtering via suffix site/dist packages where it's used.